man/systemd.exec.xml

```diff
@@ -141,8 +141,10 @@
         <term><varname>Group=</varname></term>
 
         <listitem><para>Set the UNIX user or group that the processes are executed as, respectively. Takes a single
-        user or group name, or numeric ID as argument. If no group is set, the default group of the user is used. This
-        setting does not affect commands whose command line is prefixed with <literal>+</literal>.</para></listitem>
+        user or group name, or numeric ID as argument. If not user is explicitly set the user the invoking service
```

Comment:
Hm, I was wrong. Testing indeed shows that User= works under user@0.

So some different wording should be used.